api/v1alpha1/runnerpool_types.go

```diff
@@ -44,6 +45,11 @@ type RunnerPoolSpec struct {
 	// +optional
 	SlackAgent SlackAgentConfig `json:"slackAgent,omitempty"`
 
+	// Duration until the pod is recreated
+	// +kubebuilder:default="24h"
+	// +optional
+	StaleDuration string `json:"staleDuration,omitempty"`
```

Comment:
The word "stale" will be used in two different ways in meows.
It is confusing, so please change one of them.

Reply:
Changed from `staleDuration` to `recreateDeadline`.